src/content/en/2020/mobile-web.md

```diff
@@ -8,7 +8,7 @@ authors: [spanicker, mdiblasio]
 reviewers: [obto]
 analysts: [obto]
```

Comment:
As #1817 has been merged can you add yourself here? Probably best to do a merge with `main` to address any merge conflict this creates.

```suggestion
analysts: [obto]
editors: [exterkamp]
```

Reply:
Done.